Purchases/Attribution/AttributionFetcher.swift

```diff
@@ -96,22 +96,23 @@ class AttributionFetcher {
         return true
     }
 
+    var authorizationStatus: FakeTrackingManagerAuthorizationStatus {
+        if #available(iOS 14.0.0, tvOS 14.0.0, *) {
```

Comment:
I feel like this is right (following https://developer.apple.com/documentation/apptrackingtransparency)

Reply:
yep, this is exactly right 😃 
Since we can't reference the framework directly, perhaps we can add this as a comment? i.e.: 
```suggestion
        // should match OS availability here: https://rev.cat/app-tracking-transparency
        if #available(iOS 14.0.0, tvOS 14.0.0, *) {
```

That link points to the same place, I just created it in rebrandly

Reply:
also, you can do this too if you prefer. I don't have much of a preference, but it does seem like Swift code usually handles preconditions as guards with early returns
```swift
// should match OS availability here: https://rev.cat/app-tracking-transparency
guard #available(iOS 14.0.0, tvOS 14.0.0, *)  else {
    return .notDetermined
}
return self.fetchAuthorizationStatus
```

Reply:
I think we might wanna include macOS here as well, though. I think if you don't, then macOS 10 code would actually try to call `fetchAuthorizationStatus`

Reply:
Good idea, adding!